components/scream/src/physics/p3/tests/p3_saturation_unit_tests.cpp

```diff
@@ -53,41 +114,46 @@ struct UnitWrap::UnitTest<D>::TestP3Saturation
 
     Scalar tol = (util::is_single_precision<Scalar>::value ) ? C::Tol*2 : C::Tol;
```

Comment:
Since we are actually computing the expected solution in single precision for the single precision test now, hopefully we should be able to get rid of this horribly hacky line (that I put in).

Reply:
I will remove it in my next round of changes.